clients/js/test/extensions/scaledUiAmountMint/initializeScaledUiAmountMint.test.ts

```diff
@@ -0,0 +1,87 @@
+import test from 'ava';
+import {
+  createDefaultSolanaClient,
+  generateKeyPairSignerWithSol,
+  getCreateMintInstructions,
+  sendAndConfirmInstructions,
+} from '../../_setup';
+import {
+  Account,
+  address,
+  generateKeyPairSigner,
+  isSome,
+} from '@solana/web3.js';
+import {
+  extension,
+  fetchMint,
+  getInitializeScaledUiAmountMintInstruction,
+  Mint,
+} from '../../../src';
+
+test('it initialize a mint account with an interest bearing mint extension', async (t) => {
+  const client = createDefaultSolanaClient();
+  const [authority, mint] = await Promise.all([
+    generateKeyPairSignerWithSol(client),
+    generateKeyPairSigner(),
+  ]);
+
+  const newMultiplier = 2;
+
+  // And a scaled ui amount mint extension.
+  const scaledUiAmountMintExtension = extension('ScaledUiAmountConfig', {
+    authority: authority.address,
+    multiplier: 1,
+    newMultiplierEffectiveTimestamp: BigInt(
+      Math.floor(new Date().getTime() / 1000)
+    ),
+    newMultiplier,
+  });
+
+  // When we initialize the mint account with the scaled ui amount mint extension.
+
+  // And a mint close authority extension.
```

Comment:
nit: it would be nice to keep the `Given/When/Then` comment convention for consistency with other tests. `And` prefixes are also allowed in between each to add more "paragraphs".

Ah, I've just realised that the tests for the interest bearing extension didn't follow that convention.

Reply:
Got it. Updated!

Reply:
Tysm! And thanks for updating the tests for the interest bearing extension too! 🙇